### ISSUE TITLE
UAI Reader is failing for some files.

### ISSUE THREAD

**Issue by kislayabhi** [pgmpy/pgmpy#458]
`grid3x3.uai` : http://www.cs.huji.ac.il/project/PASCAL/examples/grid3x3.uai
`grid4x4.uai` : http://www.cs.huji.ac.il/project/PASCAL/examples/grid4x4.uai

The UAI reader works fine for the `grid3x3.uai` but not for `grid4x4.uai`.

I tried the following.

```
from pgmpy.readwrite import UAI
reader = UAI.UAIReader('grid4x4.uai')
model = reader.get_model()
```

The output:

```
ValueError: Self loops are not allowed
```

I am sure that the grid4x4.uai has no self loops.

**Comment by ankurankan** [comment]
@vivek425ster Can you please have a look at this?

**Comment by ankurankan** [comment]
Fixed in #459 . @kislayabhi Please check.
